net/l3roamd/Makefile

```diff
@@ -0,0 +1,28 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=l3roamd
+PKG_SOURCE_DATE:=2016-05-06
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/freifunk-gluon/l3roamd.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=9d474a9329e307a5e71fd55d53491d93d34dda13
```

Comment:
Please update to latest. If some l3roamd commits need to be reverted for now, let's do that now.